src/Normalizer/NetworkSettingsNormalizer.php

```diff
@@ -81,6 +81,10 @@ public function denormalize($data, $class, $format = null, array $context = [])
             foreach ($data['Ports'] as $key => $value) {
                 $values_1 = [];
                 foreach ($value as $value_1) {
+                    if (null === $value) {
+                        $values[$key] = null;
+                        continue;
+                    }
```

Comment:
these lines are the ones that, applying the patch, will be created, aren't they?

Reply:
That's right

Reply:
nevermind, I forgot that the patching is done right after generation. :-D